### PR TITLE
Added content to index.html

### PR DESCRIPTION
**Pull request by Richie78321**
Added more static HTML to the About Me section, and added a section of favorite projects.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -41,6 +41,74 @@ <h3>About me</h3>
         <hr />
       </div>
 
+      <p>Beyond that, I am a pretty normal person! I enjoy playing video games, I like food more than I probably should,
+      and I have two dogs that are the best. Talking video games, I am especially fond of VR. I have been an early adopter of virtual reality for a long time now,
```
Comment (billyeh) [inline portfolio/src/main/webapp/index.html:45]: Just curious, how is line length decided for HTML? E.g. this line is a bit longer than the others.

Comment (Richie78321) [reply portfolio/src/main/webapp/index.html:45]: This is kind of arbitrary, and I just put it on different lines to keep it from getting too long. Let me take a look at Google’s HTML style guide.

Comment (Richie78321) [reply portfolio/src/main/webapp/index.html:45]: https://google.github.io/styleguide/htmlcssguide.html Seems like there isn’t a defined protocol for it beyond indenting by 4 spaces.

Comment (billyeh) [reply portfolio/src/main/webapp/index.html:45]: Yeah, thanks! https://google.github.io/styleguide/htmlcssguide.html#HTML_Formatting_Rules looks like if it improves readability then we can do it. I do like having them be a bit shorter, this is fine. 

And the block-level things are all on their own lines so think we're good. Thanks!

**Pull request merged**